spmd/tensor/ops/view_ops.py

```diff
@@ -405,6 +422,25 @@ def dim_unsqueeze(ndim: int, dim: int) -> DimMap:
     return dims[:dim] + (Singleton(),) + dims[dim:]
 
 
+def dim_reduction(
```

Comment:
I think we should probably move this to the math_ops.py and register `torch.var` there? it's a bit weird to see torch.var in view_ops.py

Reply:
makes sense -- what about i remove this for now and then add it bakc on the next PR ?

Reply:
yeah sounds good to me!